steering/CONTRIBUTION-FORMULA.md

```diff
@@ -4,13 +4,12 @@ The [steering committee charter](CHARTER.md) states "Each year, when
 Contribution Seat terms begin, the Steering Committee will vote on an exact
 formula and procedure for determining allocation for the following year."
 
-For the February 2021 to January 2022 term, the contribution formula shall be
-**merged pull requests** attributed to a Contributing Company,
+For the February 2022 to January 2023 term, the contribution formula shall be
+**contributions** (review, comment, commit, issue or PR on GitHub) attributed to a Contributing Company,
```

Comment:
Can you point at the script we're using to calculate (since it diverges from the github contribution stat)?

Reply:
Also interested in a diff of the results of old vs new methodology to see if there are any meaningful differences

Reply:
Multi dimensions could help motivate the community activities. I think it is better than before.

Reply:
> Also interested in a diff of the results of old vs new methodology to see if there are any meaningful differences

<img width="1272" alt="Screen Shot 2022-04-01 at 2 22 42 PM" src="https://user-images.githubusercontent.com/132510/161176612-336976fd-1b0b-4b44-b3b4-6708e8da33be.png">

This isn't on the exact same time period, but I think it produces an acceptably similar outcome to the current formula, while accurately reflecting contribution in other areas of the project.

Reply:
This is a hyperthesis, huawei will keep the contributions until 1.31, 2023.

Reply:
I would note that 80% of "contributions" are from comments. A comment has much less value than something like a PR or an issue (for example, I just made a contribution by writing this). Especially with things like `/retest`, etc.

The end result is almost the same as PRs, so it's not a big concern in the current allocations - just something that could easily be gamed if someone wanted to.

No metric is perfect, so I'm not suggesting we change it

Reply:
This was discussed.  Sustained low-effort comments in order to try and game the system would be addressed through the Code of Conduct, and could result in people being removed from the community.

I like the fact that right now contributions effectively have to be approved by another community member (as we're counting merged PRs that passed review), but we are trying to incentivize PR review also.

We could remove comments and see the outcome?  Let's give that a go...

Reply:
Assume pr review comments and the comments here are not the same. But think of comments under an issue., which may be of great help for end users

Reply:
<img width="1275" alt="Screen Shot 2022-04-01 at 3 01 47 PM" src="https://user-images.githubusercontent.com/132510/161180032-3dbf2a3b-fc46-4518-ba5b-8c965ac84c86.png">

Same result, slightly different way of getting there.

Reply:
Turns out what we're seeing measured is "PR review comments", not "PR reviews".  Devstats doesn't measure reviews.  See https://github.com/cncf/devstats/issues/341

Reply:
Personally i think "PR review comments" may be more fair, as one pr can be super huge and one could be small, the review comments number should be different based on the effort